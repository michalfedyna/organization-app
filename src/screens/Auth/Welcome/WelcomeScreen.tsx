import React from 'react';
import Layout from '@components/Layout';

const WelcomeScreen = () => {
  return <Layout.Container />;
};

export default WelcomeScreen;
