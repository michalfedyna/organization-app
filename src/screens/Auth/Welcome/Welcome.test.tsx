import React from 'react';
import {test, expect} from '@jest/globals';
import renderer from 'react-test-renderer';

import WelcomeScreen from '@screens/Auth/Welcome/WelcomeScreen';

test('WelcomeScreen renders correctly', () => {
  const tree = renderer.create(<WelcomeScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
