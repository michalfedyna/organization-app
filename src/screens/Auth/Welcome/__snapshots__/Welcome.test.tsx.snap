// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen renders correctly 1`] = `
<View
  style={
    {
      "flexDirection": "column",
    }
  }
/>
`;
